internal/impl/kafka/enterprise/redpanda_migrator_bundle_output.tmpl.yaml

```diff
@@ -119,8 +105,7 @@ mapping: |
                     cases:
                       - check: '@fallback_error == "request returned status: 422"'
                         output:
-                          # TODO: Use a DLQ
-                          drop: {}
+                          drop: {} # TODO: Use a DLQ?
```

Comment:
We want to drop these messages (and potentially forward them to a DLQ?), because they indicate that the provided schema already exists in the destination.